src/main/resources/org/broadinstitute/barclay/helpTemplates/wdlToolTemplate.wdl.ftl

```diff
@@ -132,7 +132,15 @@ task ${name} {
   output {
     <@defineTaskOutputs heading="Task Outputs" outputs=runtimeOutputs/>
   }
- }
+  <#if localizationOptional?size != 0>
+
+  parameter_meta {
+    <#list localizationOptional as optLoc>
```

Comment:
Something to think about - should we move the comments in the header describing each input variable to the parameter meta section since that's what it's for?

Reply:
Hm, yeah, that probably makes sense.

Reply:
I added a param_meta section to both the workflow and task, which cromwell and womtool both seem to be happy with. I also left the comments at the top since it seems harmless and they're easy to scan there. Let me know if you disagree.